pyrometheus/codegen/python.py

```diff
@@ -304,6 +306,20 @@ def get_mixture_internal_energy_mass(self, temperature, mass_fractions):
         emix = self.get_mass_average_property(mass_fractions, e0_rt)
         return self.gas_constant * temperature * emix
 
+    def get_specific_heat_ratio(self, temperature, mass_fractions):
+        cp0_r = self.get_species_specific_heats_r(temperature)
+        cpmix = self.get_mass_average_property(mass_fractions, cp0_r)
+        sum_Yk_Wk = sum([
+            mass_fractions[i]
+            * self.inv_molecular_weights[i]
+            for i in range(self.num_species)])
+        return cpmix / (cpmix - sum_Yk_Wk)
+
+    def get_soundspeed(self, temperature, mass_fractions):
+        gamma = self.get_specific_heat_ratio(temperature, mass_fractions)
+        R = self.get_specific_gas_constant(mass_fractions)
```

Comment:
Two issues:
- Variable name not sufficiently descriptive;
- Capital R is not allowed per Flake8.

Suggested change: `mix_gas_constant`, `gas_constant_mix`, or even `gas_constant_specific`.